src/data/BPDA.ts

```diff
@@ -0,0 +1,427 @@
+export const BPDA: MapGeoJsonData = {

```

Comment:
This is fine for now, but as a heads up, based off of last week's discussions the idea is to eventually have this just straight up .json, so there is no variable declaration, exporting, etc. and the file starts and end with curly braces.